src/bin/common/solving_loop.ml

```diff
@@ -358,9 +358,9 @@ let main () =
       if Stdlib.(Options.get_sat_solver () = Tableaux) then
         Options.set_interpretation ILast
       else Printer.print_wrn "%a The generation of models is not supported \
-                              for the current SAT solver. Please choose the \
+                              for the SAT solver %a. Please choose the \
                               SAT solver Tableaux."
```

Comment:
Rather than ask the user to choose the Tableaux sat solver, we should recommend using `--produce-models`. This will effectively switch to the Tableaux solver, but at least it is forward-compatible --- once we support model generation with other solvers, the same command line will keep working as intended without forcing the use of the Tableaux solver.